src/components/settings-menu/WalletNotificationsSettings.tsx

```diff
@@ -36,15 +101,20 @@ const WalletNotificationsSettings = () => {
           }
         />
       </Menu>
-      {notificationsAllowed && (
+      <Animated.View style={animatedStyle}>
         <Menu>
           <MenuItem
             disabled
             hasSfSymbol
             leftComponent={
               <MenuItem.TextIcon colorOverride={colors.appleBlue} icon="􀈟" />
             }
-            rightComponent={<Switch />}
+            rightComponent={
+              <Switch
+                value={notifications?.topics[NotificationTopic.SENT]}
+                onValueChange={() => toggleTopic(NotificationTopic.SENT)}
```

Comment:
useCallback?

Reply:
you can use it directly here so it will make it more succinct

Reply:
already using it with `toggleTopic` and `onValueChange` needs to return `() => void` unless you meant something else?

Reply:
It's ok